### DISCUSSION TITLE
[Feature] Make people data per asset available offline

### DISCUSSION THREAD

**Discussion by dev-ritik** [immich-app/immich#14754] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Right now the mobile client fetches people for a given asset once we open the info panel. However, there is no data when the device is offline. Google Photos shows this data even when working offline for local pictures.

### Possible implementation
We will need bulk APIs to get people's asset mapping for offline assets to get this working. Looking at the code, I think there is no API right now to give asset people data in bulk. The [delta sync](https://immich.app/docs/api/get-delta-sync/) and the full sync for user API do have optional people in response; however, they are never sent.

Once such API is available, we can keep people data offline in mobile db and render appropriately

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4466
